packages/dx-react-grid-material-ui/src/templates/table-group-row-cell.jsx

```diff
@@ -1,15 +1,12 @@
 import React from 'react';
 import PropTypes from 'prop-types';
-
 import ChevronRight from 'material-ui-icons/ChevronRight';
 import ExpandMore from 'material-ui-icons/ExpandMore';
-
-import {
-  TableCell,
-} from 'material-ui';
-
+import { TableCell } from 'material-ui/Table';
 import { withStyles } from 'material-ui/styles';
 
+const ENTER_KEY_CODE = 13;
```

Comment:
Can we extract key code const(s) to a separate file and share it between bootstrap & MUI?